crates/cubecl-hip/src/runtime.rs

```diff
@@ -107,6 +107,8 @@ fn create_client<M: WmmaCompiler<HipDialect<M>>>(
         // This is a guess - not clear if ROCM has a limit on the number of bindings,
         // but it's dubious it's more than this.
         max_bindings: 1024,
+        // Minimum supported
+        max_shared_memory_size: 49152,
```

Comment:
Is the comment right?

Reply:
it used to just be hardcoded to that for both CUDA and HIP, so I assume it was the minimum supported? I now check the actual minimum for CUDA (since it's almost always higher than 48k), but I don't fully understand the HIP code and can't really test if my code works so I used the hardcoded constant from before.